manifests/kustomize/base/pipeline/ml-pipeline-apiserver-role.yaml

```diff
@@ -13,6 +13,12 @@ rules:
   - get
   - list
```

Comment:
https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/mysql.yaml#L46

would you help remove the line in one shot?

Reply:
it's OK if no test for now.

Reply:
so does MINIO.
I think MySQL and MINIO won't hit the createRun to allow you remove it.
https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/minio.yaml#L46

Reply:
Thanks for pointing them out. I prefer a separate one because this is complex enough.
Will do that shortly after this.

Reply:
OK, or feel free to leave it to me if multi-user is busy